### ISSUE TITLE
Qubes-Whonix

### ISSUE THREAD

**Issue by bluesteal** [vmonaco/keystroke-obfuscation#3]
Is this version of kloak compatible with Qubes-Whonix?

**Comment by adrelanos** [comment]
No, but that's an issue caused by Qubes. (Not Whonix.) The issue is tracked here:

https://github.com/QubesOS/qubes-issues/issues/1850
